compiler/src/dmd/dtemplate.d

```diff
@@ -7561,7 +7561,7 @@ extern (C++) class TemplateInstance : ScopeDsymbol
                 //printf("\t staticIfDg on '%s %s' in '%s'\n",  s.kind(), s.toChars(), this.toChars());
                 if (!s.isStaticIfDeclaration())
                 {
-		    //s.dsymbolSemantic(sc2);
+                    //s.dsymbolSemantic(sc2);
```

Comment:
Unrelated, own commit

Reply:
That came about because of my checkin process converts all tabs to spaces, and apparently there were tabs on that line. Being a bit of trivia, there's no issue here.